src/tests/Mission.test.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import Mission from '../components/Mission';
+import { joinMission, leaveMission } from '../redux/missions/missionSlice';
+
+const mockStore = configureStore([]);
+
+describe('Mission component', () => {
+  let store;
+  let missions;
+
+  beforeEach(() => {
+    missions = [
+      {
+        mission_id: '1',
+        mission_name: 'Mission 1',
+        description: 'Mission 1 description',
+        website: 'https://mission1.com',
+        wikipedia: 'https://en.wikipedia.org/wiki/Mission1',
+        twitter: 'https://twitter.com/mission1',
+        reserved: false,
+      },
+      {
+        mission_id: '2',
+        mission_name: 'Mission 2',
+        description: 'Mission 2 description',
+        website: 'https://mission2.com',
+        wikipedia: 'https://en.wikipedia.org/wiki/Mission2',
+        twitter: 'https://twitter.com/mission2',
+        reserved: true,
+      },
+    ];
+
+    store = mockStore({
+      mission: {
+        missions,
+        isLoading: false,
+      },
+    });
+
+    store.dispatch = jest.fn();
+  });
+
+  test('renders mission data', () => {
+    render(
+      <Provider store={store}>
+        <Mission />
+      </Provider>,
+    );
```

Comment:
- [ ]  You have done great job to impliment the tests as suggested by provided by previous reviewers, but there some issue I am facing while I am running them.  check the image to look the issue and fix it . 
![image](https://github.com/davidolanrewaju/Space-Traveler-Hub/assets/43877689/567581ef-2b7c-4aa7-bfa5-e224554a503f)

Reply:
@ishimwezachee please can you explain better what the issue is and an idea or a guide on how to resolve it